test-utils/src/lib.rs

```diff
@@ -302,6 +303,11 @@ impl CoreWfStarter {
         self
     }
 
+    pub fn no_remote_activities(&mut self) -> &mut Self {
```

Comment:
I don't love how this ended up everywhere, but I don't want to hold things up on getting rid of it either, so I can see if I can do some dedupe that combines options & wf/act registration all in one go so this gets automatically set when needed, but I don't wanna block the PR on it.

Reply:
Agree, don't love it either but I didn't want this PR to get out of hand.